pkg/operator/ceph/csi/spec.go

```diff
@@ -307,11 +315,51 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		Namespace: r.opConfig.OperatorNamespace,
 	}
 
-	tp.DriverNamePrefix = fmt.Sprintf("%s.", r.opConfig.OperatorNamespace)
+	if strings.HasSuffix(tp.DriverNamePrefix, ".") {
```

Comment:
It is acceptable to have a prefix with a dot in the middle, right? Just to confirm an additional dot doesn't have any significance to the provisioner name. For example: `my.prefix`

Reply:
Yes thats correct.